src/boxFunctions/boxEnclosingPoints.ts

```diff
@@ -0,0 +1,33 @@
+import { IVec, boxReset } from "..";
+
+/**
+ * Computes the smallest bounding box that contains all of the provided points.
+ *
+ * If the provided array is empty, this method returns a box with all values set
+ * to `NaN`.
+ *
+ * @param points the points to contain
+ */
+export function boxEnclosingPoints(...points: IVec[]) {
+  let minX: number = NaN;
+  let maxX: number = NaN;
+  let minY: number = NaN;
+  let maxY: number = NaN;
```

Comment:
Hmm, could you start with these values at min `Infinity` and max `-Infinity`, and remove the isNan() checks in the loop? The empty box is then `[Infinity, Infinity, -Infinity, -Infinity]` which I'm also happy with.